cli/cli.go

```diff
@@ -178,8 +178,6 @@ func SetupCLI(args []string) error {
 func (runOptions *runOptionsType) commonCLIHandler(
 	ctx *cli.Context) (*conf.MenderConfig, error) {
 
-	fmt.Println("runOptions datastore: ", runOptions.dataStore)
```

Comment:
S: I know a bit anal, but could we make these actual log messages, and have them at `debug` or maybe even `trace`?

Reply:
Okey. I'll merge as-is to and submit a follow-up PR.

Reply:
See:
* https://github.com/mendersoftware/mender-setup/pull/22